Product.wxs

```diff
@@ -24,6 +24,17 @@
       <!-- TODO: Consider using a separate feature for the symbols. This folder might
            include both WNBD and Ceph symbols. -->
       <ComponentGroupRef Id="SymbolsComponentGroup" />
+      <!-- Force delete mappings on uninstall-->
+      <Component Id="CleanRbdWnbdMappings"
+                 Directory="TARGETDIR"
+                 Guid="{280201D5-35E7-45D6-83B9-577EA1597BB5}"
+                 KeyPath="yes">
+        <RegistryKey Id="CleanupRegistry"
+                     Root="HKLM"
+                     Key="SYSTEM\CurrentControlSet\Services\rbd-wnbd"
```

Comment:
I think we should also remove the WNBD key: ``SYSTEM\CurrentControlSet\Services\rbd-wnbd``

https://github.com/cloudbase/wnbd/blob/master/include/wnbd_ioctl.h#L4

Reply:
`HKLM\SYSTEM\CurrentControlSet\Services\wnbd` removed as well, thanks for the pointers.

Reply:
Awesome, thanks!